distro/pom.xml

```diff
@@ -49,6 +49,11 @@
     <ordertemplates.version>1.0.2</ordertemplates.version>
     <patientflags.version>3.0.7</patientflags.version>
     <o3forms.version>2.3.0-SNAPSHOT</o3forms.version>
+    <patientflags.version>3.0.5</patientflags.version>
```

Comment:
Did you intentionally change the patient flags and o2forms modules?

Reply:
Removed, slipped somehow during conflict resolution.